<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="refSlider">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="refGraph">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>401</width>
      <height>291</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="userGraph">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>401</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="userSlider">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>360</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="refPlayTime">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>390</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="userPlayTime">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>390</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="userpausebutton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>390</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="userplaybutton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>390</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QLabel" name="reffreqlabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="reffreqtextbox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>430</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="refHzlabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>430</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Hz</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="userfreqtextbox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>430</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="userHzlabel">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>430</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Hz</string>
    </property>
   </widget>
   <widget class="QLabel" name="userfreqlabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>430</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="locktimecheckbox">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>390</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Lock time with ref</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recordbutton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Record</string>
    </property>
   </widget>
   <widget class="QLabel" name="refsignallabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Reference Signal</string>
    </property>
   </widget>
   <widget class="QLabel" name="usersignallabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>User Signal</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="reftablewidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>401</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="usertablewidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>470</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuImport">
    <property name="title">
     <string>Import</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionImport_Ref_File"/>
    <addaction name="actionImport_User_File"/>
   </widget>
   <addaction name="menuImport"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionImport_Ref_File">
   <property name="text">
    <string>Import Ref File</string>
   </property>
  </action>
  <action name="actionImport_User_File">
   <property name="text">
    <string>Import User File</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
